Expression to identify if the pipeline is remote or inline (#3711)
harness/developer-hub: docs/platform/variables-and-expressions/harness-variables.md

The pipeline level delegate selectors selected via runtime input.  

### <+pipeline.storeType>

> If the pipeline is stored in Harness, the expression resolves to inline. If the pipeline is stored in a Git repository, the expression resolves to remote.

Reviewer: maybe we add formatting around "inline" and "remote" so it's easier to understand?

Reviewer: Also we should be adding formatting around the "null" in the second and third entries. 

Other than these, this looks good. Thanks, @sarthakkasat

> If the pipeline is stored in Harness, the expression resolves to inline. If the pipeline is stored in a Git repository, the expression resolves to remote.

Reviewer 2: ```suggestion
If the pipeline is stored in Harness, the expression resolves to `inline`. If the pipeline is stored in a Git repository, the expression resolves to `remote`.
```

> If the pipeline is stored in harness the expression will resolve to Inline, if the pipeline is stored in Git repository the expression will resolve to remote.

Reviewer 2: ```suggestion
If the pipeline is stored in Harness, the expression resolves to inline. If the pipeline is stored in a Git repository, the expression resolves to remote.
```

### <+pipeline.repoName>

> For remote pipelines, the expression resolves to the Git repository name. For inline pipelines, the expression resolves to null.

Reviewer 2: ```suggestion
For remote pipelines, the expression resolves to the Git repository name. For inline pipelines, the expression resolves to `null`.
```

> For Remote pipeline the expression will resolve to Git repository name. For Inline pipeline the expression will resolve to null.

Reviewer 2: ```suggestion
For remote pipelines, the expression resolves to the Git repository name. For inline pipelines, the expression resolves to null.
```

### <+pipeline.branchName>

> For remote pipelines, the expression resolves to the Git branch where the pipeline exists. For inline pipelines, the expression will resolve to null.

Reviewer 2: ```suggestion
For remote pipelines, the expression resolves to the Git branch where the pipeline exists. For inline pipelines, the expression resolves to `null`.
```

> For Remote pipeline the expression will resolve to Git branch in which pipeline is present. For Inline pipeline the expression will resolve to null.

Reviewer 2: ```suggestion
For remote pipelines, the expression resolves to the Git branch where the pipeline exists. For inline pipelines, the expression will resolve to null.
```
